### DISCUSSION TITLE
Welcome to Discussions

### DISCUSSION THREAD

**Discussion by gsuttie** [CloudFamily/CloudFamily#34] [General]
Welcome to the discussions area of the CloudFamily repository - we can put in questions and link them to issues etc - and generate discussions about the code etc.

You can reference issues etc within this area

**Comment by PixelRobots** [comment]
Hello!

**Reply by gsuttie** [reply]
Hey @PixelRobots im gonna tag @samsmithnz  see if he joins in

**Reply by samsmithnz** [reply]
Hi! Question about using GitHub with automation. Anyone know of a good C# SDK I can use to upload files to a repo with C#?

**Reply by gsuttie** [reply]
Not found much on that tbh - other than this https://www.codeproject.com/Articles/1277348/Using-Csharp-Code-to-Access-the-GitHub-API

**Reply by samsmithnz** [reply]
It's a tough one. Playing with the idea of dynamically generating YAML and then uploading it to GitHub. It's a weird situation. How is everyone? Day 14 of isolation here.

**Reply by samsmithnz** [reply]
Happy to help with the actions for this site too @PixelRobots

**Reply by AlanFlorance** [reply]
hello all, hope everyones safe and well
